src/mudlet.cpp

```diff
@@ -1575,32 +1575,40 @@ void mudlet::slot_timer_fires()
     if (!pQT) {
         return;
     }
-    if (mTimerMap.contains(pQT)) {
-        TTimer* pTT = mTimerMap[pQT];
+
+    // Pull the Host* and TTimer::id from the properties:
```

Comment:
Um, it was not possible to store the pointer value as a `QString` so this should say Host name now.

Reply:
Fixed.